doc/DependencyInjection.md

```diff
@@ -2,11 +2,13 @@
 
 We use [GetIt](https://pub.dev/packages/get_it) for any IoC related work.
 
+For more documentation on dependency injection, read the references listed at the bottom.
+
 ## Registering
 
-- You can register a singleton service using `GetIt.I.registerSingleton<IService>(ServiceImplementation())`. This service will be accessible from anywhere.
+- You can register a singleton service using `GetIt.I.registerSingleton<IService>(ServiceImplementation())`. This service will be shared by every user.
```

Comment:
What is the change in wording supposed to say?

Reply:
I'm just wondering

Reply:
I have no idea this is from thomas's commit that im trying to remove since I based my branch on his